docs/1 Introduction/1 Introduction.md

```diff
@@ -22,25 +20,23 @@ Many projects already showcase the Tendermint BFT Consensus Engine and the Cosmo
 
 ## Projects building with Starport
 
-* https://github.com/Sifchain/sifnode
-* https://github.com/crypto-org-chain/chain-main (initialized with Starport: https://github.com/crypto-org-chain/chain-main/commit/37b2ecb49a9aae7c581270a4f2dbecfcd8e8a6e9)
-* Interchain accounts demo: https://github.com/chainapsis/cosmos-sdk-interchain-account
-* https://github.com/lazyledger/lazyledger-app
-* https://github.com/wangfeiping/flares
-* https://github.com/EG-easy/affondra
-* https://github.com/chantmk/Finding-imposter
-* https://github.com/umee-network/umee
-* https://github.com/tendermint/spn
-* https://github.com/CosmosContracts/Juno
+* [Sifnode](https://github.com/Sifchain/sifnode)
+* [crypto.org chain](https://github.com/crypto-org-chain/chain-main) ([initialized with Starport](https://github.com/crypto-org-chain/chain-main/commit/37b2ecb49a9aae7c581270a4f2dbecfcd8e8a6e9))
+* [Interchain accounts demo](https://github.com/chainapsis/cosmos-sdk-interchain-account)
+* [Lazy Ledger](https://github.com/lazyledger/lazyledger-app)
+* [Flares](https://github.com/wangfeiping/flares)
+* [Affondra](https://github.com/EG-easy/affondra)
```

Comment:
looks like Affondra is a hackathon project for trading NFT items, with many typos on the README page (I see now that many of these projects were class projects or hackathon projects)

Reply:
Two of them are hackaton projects, but I think it's still ok to showcase them here. After all Starport should be (and is) useful for teams participating in hackathons.